src/api/editor.ts

```diff
@@ -51,7 +51,7 @@ const getSqlResult = (code: string) => {
 }
 
 const getScriptsTable = (db: string) => {
-  return axios.post(sqlUrl, makeSqlData(`select * from scripts where schema = '${db}'`))
+  return axios.post(sqlUrl, makeSqlData(`select * from public.scripts where schema = '${db}'`))
```

Comment:
```suggestion
  return axios.post(sqlUrl, makeSqlData(`select * from scripts where schema = '${db}'`))
```

since `schema = ${db}` is still there, `public` is not appropriate

Reply:
@shuiyisong this is a temporary solution when scripts table is stored in `public` schema. It's going to be moved to a system schema as discussed in https://github.com/GreptimeTeam/greptimedb/issues/665 and partially implemented in https://github.com/GreptimeTeam/greptimedb/pull/868

Here we use fully qualified table name `public.scripts` to ensure it can be accessed from any schema. And script editor feature won't be available on cloud until database 0.2

Reply:
got it. maybe a TODO is needed here 😆